packages/devtools/lib/src/framework/framework.dart

```diff
@@ -121,6 +124,17 @@ class Framework {
       mainElement.add(screenContent);
 
       _screenContents[current] = screenContent;
+
```

Comment:
Add a comment that this is stopping event listeners within the screen from getting the resize event not stop event listeners higher up in the tree. I misunderstood that on first read. Give Plotly as a concrete example of why this is needed.